test/verify/test_atanh.cpp

```diff
@@ -23,20 +23,24 @@
  */
 
 #include "verify_program.hpp"
+#include <migraphx/float8.hpp>
+#include <migraphx/half.hpp>
 #include <migraphx/program.hpp>
 #include <migraphx/generate.hpp>
 #include <migraphx/make_op.hpp>
 
-struct test_atanh : verify_program<test_atanh>
+template <typename CType>
+struct test_atanh : verify_program<test_atanh<CType>>
 {
     migraphx::program create_program() const
     {
         migraphx::program p;
-        auto* mm = p.get_main_module();
-        migraphx::shape s{migraphx::shape::float_type, {16}};
+        auto* mm                      = p.get_main_module();
+        migraphx::shape::type_t dtype = migraphx::shape::get_type<CType>();
+        migraphx::shape s{dtype, {16}};
         auto x       = mm->add_parameter("x", s);
-        auto min_val = mm->add_literal(-0.95f);
-        auto max_val = mm->add_literal(0.95f);
+        auto min_val = mm->add_literal(migraphx::literal{migraphx::shape{dtype}, {-0.95f}});
+        auto max_val = mm->add_literal(migraphx::literal{migraphx::shape{dtype}, {0.95f}});
```

Comment:
Same as my above comment, do we need to drop the "f" here for these decimal values and do  a double->fp8 or should the conversion be from float -> fp8 or it doesn't matter?

Reply:
Doesn't matter.